src/Bundle/test/config/packages/test/sylius_mailer.yaml

```diff
@@ -1,4 +1,5 @@
 sylius_mailer:
+    sender_adapter: sylius.email_sender.adapter.symfony_mailer
```

Comment:
WOn't it be better to make it work without this line?

Reply:
I believe it does? 😅 It's a default value, as far as far know, maybe this line is not needed 🖖